src/main/java/org/hyperledger/fabric_ca/sdk/HFCAClient.java

```diff
@@ -1614,10 +1616,35 @@ private void setUpSSL() throws InvalidArgumentException {
                         }
                     }
                 }
+                SSLContextBuilder sslContextBuilder = SSLContexts.custom()
+                        .loadTrustMaterial(cryptoPrimitives.getTrustStore(), null);
 
-                SSLContext sslContext = SSLContexts.custom()
-                        .loadTrustMaterial(cryptoPrimitives.getTrustStore(), null)
-                        .build();
+                String tlsClientKeyFile = properties.getProperty("tlsClientKeyFile");
+                String tlsClientCertFile = properties.getProperty("tlsClientCertFile");
+
+                byte[] tlsClientKeyAsBytes = (byte[]) properties.get("tlsClientKeyBytes");
+                byte[] tlsClientCertAsBytes = (byte[]) properties.get("tlsClientCertBytes");
+
+                if (tlsClientCertFile != null && tlsClientKeyFile != null) {
+                    tlsClientKeyAsBytes = Files.readAllBytes(Paths.get(tlsClientKeyFile));
+                    tlsClientCertAsBytes = Files.readAllBytes(Paths.get(tlsClientCertFile));
+                }
+
+                if (tlsClientKeyAsBytes != null && tlsClientCertAsBytes != null) {
+                    Certificate tlsCleintCertificate = new CryptoPrimitives().bytesToCertificate(tlsClientCertAsBytes);
```

Comment:
I'm being very picky but it's worth getting it right... `tlsCleintCertificate` should be `tlsClientCertificate`